aries-mobile-tests/pageobjects/bc_wallet/credential_details.py

```diff
@@ -24,7 +24,12 @@ def on_this_page(self):
 
     def select_back(self):
         if self.on_this_page():
-            self.find_by(self.back_locator).click()
+
+            if self.driver.desired_capabilities['platformName'] == 'Android':
```

Comment:
This works of course, but the platform is already set from capabilities on the basepage, so self.current_platform works. This will not postpone the merge though.